### PR TITLE
web-translate-it 1.0.0

### PR DESCRIPTION
**Pull request by mAu888**
This adds a new step that installs the [Web Translate It](https://webtranslateit.com) gem and performs a `wti pull` operation.

### CODE REVIEW

**Comment by viktorbenei** [comment]
Hi, there are a couple of minor issues with the step.yml - did you use the latest `bitrise CLI` version (1.3.3)?

We'll do a quick review and list the issues, but please make sure that you use the latest CLI version for step sharing, as we frequently add new notes / guides / validations to it, so that when you share your step we just have to accept it :)

To upgrade your `bitrise CLI` version just run: `brew update && brew upgrade bitrise`

**Comment by viktorbenei** [comment]
Actually the related audit notes are not yet available in the current CLI version, the latest stable (1.3.3) did not print any issue. Sorry, we forgot that those are not yet released validation/audit notes :)

File: steps/web-translate-it/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+type_tags:
+- translation
+- localization
+- xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- WTI_API_KEY: null
```
Comment (godrei) [inline steps/web-translate-it/1.0.0/step.yml]: can you please change it to lower case? it would better fit to bitrise steplib.

**Comment by mAu888** [comment]
Cool. But you were right though, I was using `1.3.2`. Any need to re-share the step?

File: steps/web-translate-it/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  opts:
+    description: If no `.wti` file can be located in the $WTI_WORKING_DIRECTORY (defaults
+      to current working directory), the step creates a `.wti` file based on the $WTI_API_KEY
+      you provide. If neither a `.wti` file nor an $WTI_API_KEY are given, the step
+      fails.
+    is_required: false
```
Comment (godrei) [inline steps/web-translate-it/1.0.0/step.yml]: are you sure it is not required? on bitrise virtual machines no .wti file will exist.

File: steps/web-translate-it/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    summary: Find the API key on webtranslateit.com in your project settings.
+    title: Web Translate It API Key
+- WTI_WORKING_DIRECTORY: null
```
Comment (godrei) [inline steps/web-translate-it/1.0.0/step.yml]: can you please change it to lower case?

**Comment by viktorbenei** [comment]
Unfortunately yes, to conform to rules which are not yet audited during the share in the current 1.3.3 version.

See the things @godrei listed

File: steps/web-translate-it/1.0.0/step.yml (hunk 2 above)
Comment (mAu888) [inline steps/web-translate-it/1.0.0/step.yml]: Yes, the file can be checked into the repository itself. That's how we do it at the moment. You can have some hooks in that `.wti` file. Only if you don't check in the file the step autocreates one by using the API key.

**Comment by mAu888** [comment]
What's the correct way of re-sharing? Remove current `1.0.0` tag and start the process from scratch or do a `1.0.1`?

**Comment by viktorbenei** [comment]
Btw you don't have to change this version, we'll just simply not accept it, but you don't have to delete it from GitHub. Just fix the things, create a new release and share the new version. It's not required at all that the first version you share is `1.0.0`, it can be `1.0.1` ;)

**Comment by mAu888** [comment]
Hm, ok that apparently did not work (the 1.0.0 is still part of my steplib). How to proceed here? Manually remove the `1.0.0` folder and push to my steplib fork and create a new PR?

**Comment by viktorbenei** [comment]
The easiest way is to just delete the branch in your fork: https://github.com/mAu888/bitrise-steplib/branches and share it again

**Pull request closed without merging**